models/discussionModel.js

```diff
@@ -0,0 +1,50 @@
+const { Schema, model } = require("mongoose");
+const slugify = require("slugify");
+
+const discussionSchema = Schema(
+  {
+    id: {
+      type: String,
+      required: true,
+      unique: true,
+    },
+    user_id: {
+      type: String,
+      required: true,
+    },
+    title: {
+      type: String,
+      required: [true, "Please Add a Discussion title"],
+    },
+    body: {
+      type: String,
+      required: [true, "Please Add a body text"],
+    },
+    attachments: {
```

Comment:
this should be able to take a list of files, like an array of files

Reply:
I totally forgot, I wanted to change it to array before I push, will fix it now